hbase-common/src/main/java/org/apache/hadoop/hbase/ChoreService.java

```diff
@@ -173,9 +173,7 @@ public synchronized boolean scheduleChore(ScheduledChore chore) {
    * @param chore The Chore to be rescheduled. If the chore is not scheduled with this ChoreService
    *          yet then this call is equivalent to a call to scheduleChore.
    */
-  private synchronized void rescheduleChore(ScheduledChore chore) {
-    if (chore == null) return;
-
```

Comment:
This is a private method.  The public methods that call this are already synchronized and already ensure that 'chore' is not null.